tutorials/platform/android/android_plugin.rst

```diff
@@ -49,8 +49,12 @@ with the following caveats:
 Building an Android plugin
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**Prerequisite:** `Android Studio <https://developer.android.com/studio>`_ is strongly recommended as the IDE to use to create Android plugins.
-The instructions below assumes that you're using Android Studio.
+**Prerequisite:** Android Studio is strongly recommended as the IDE to use to create Android plugins. Currently the latest version of Android Studio
+is not compatible with Godot because it requires JDK 17 and Godot requires JDK 11.
+(see the release notes of `JDK 17 <https://developer.android.com/build/releases/gradle-plugin#jdk-17-agp>`_ and `AGP 8.0 <https://developer.android.com/build/releases/gradle-plugin#8-0-0>`_).
+**Android Studio Electric Eel | 2022.1.1 Patch 2** is the last release to support JDK 11 out of the box. It can be downloaded from the `Android Studio Archive <https://developer.android.com/studio/archive>`_.
+Alternatively, you can try adapting the `Godot Google Play Billing <https://github.com/godotengine/godot-google-play-billing>`_ plugin.
+The instructions below assume that you're using Android Studio.
```

Comment:
I'm not sure what this means.. The `Google Play Billing` is not a template / starter project and not designed to be so, so I don't see how it's relevant as an alternative to Android Studio.

Reply:
You're right it's not a template. But it can easily be used as one. All I did was change a few settings & names and was able to use it for my Android plugin.

Reply:
We can create a separate starter / template project and link it to the documentation if that's helpful.